TheWarWithin/DeathKnightFrost.lua

```diff
@@ -1969,6 +1957,30 @@ spec:RegisterOptions( {
     package = "Frost DK",
 } )
 
+--[[ Estimation of whether or not random RP gains can happen, in an attempt to smooth out changing recommendations
```

Comment:
?

Reply:
A way to smooth out rotation jumpiness, but it needs more iteration. May come back to it once the groundwork is pushed through.

Basically breath window has a bunch of RP thresholds, and a random +3 or +5 that can't be predicted can flip-flop a recommendation, and between the 2 recs it really doesn't matter a whole lot. The miniscule dps increase isn't worth the jarring rotation for the user.

![alt text](image.png)

Reply:
![image](https://github.com/user-attachments/assets/45687b66-5f9a-48e2-9aaf-353ab7b895d0)